std_msgs/Header header
float32[] angle
float32[] velocity
float32[] current
float32[] temperature
float32[] pred_exte
float32[] pred_priv

float32[] body_vel
float32[] body_quat
float32[] body_gyro
float32[] body_acc
float32[] commands
float32[] heightscan
float32[] wrench

float32[] ref_angle
float32[] ref_velocity
float32[] ref_kp
float32[] ref_kd
float32[] ref_torque
